File: AutoVoiceServer/arbitration/src/main/java/com/autovoice/server/arbitration/DecisionSink.java
package com.autovoice.server.arbitration;

import com.autovoice.server.contracts.DecisionEntry;

/** 决策日志出口：仲裁器把每次收敛决策写到这里（shared/protocol.md §6）。 */
public interface DecisionSink {

    void log(DecisionEntry e);
}

File: AutoVoice/voice-core/src/main/kotlin/com/autovoice/voicecore/arbiter/DecisionSink.kt
package com.autovoice.voicecore.arbiter

import com.autovoice.voicecore.DecisionEntry

/**
 * 端侧仲裁器决策日志出口：每次竞速收敛写一条 [DecisionEntry]。
 */
fun interface DecisionSink {
    fun onDecision(entry: DecisionEntry)
}
